lib/serrano.rb

```diff
@@ -234,6 +234,8 @@ def self.works(ids: nil, query: nil, filter: nil, offset: nil,
                  select: nil, options: nil, verbose: false, cursor: nil,
                  cursor_max: 5000, **args)
 
+    assert_valid_filters(filter) if filter
```

Comment:
filters can be used on other api routes if `works=true`, so can you also add this line to: `members`, `prefixes`, `funders`, and `journals`?